Include/longobject.h

```diff
@@ -212,6 +212,7 @@ PyAPI_FUNC(PyObject *) _PyLong_GCD(PyObject *, PyObject *);
 #ifndef Py_LIMITED_API
 PyAPI_FUNC(PyObject *) _PyLong_Rshift(PyObject *, size_t);
 PyAPI_FUNC(PyObject *) _PyLong_Lshift(PyObject *, size_t);
+PyAPI_FUNC(PyObject *) _PyLong_AddInt(PyLongObject *, int);
```

Comment:
Should this API public?
If no, please move it to `Include/internal`.
If yes, how about using `Py_ssize_t` instead of `int`?

Reply:
It should not be public. I will move it. I am still not used to what goes where now that we've got several different subdirectories of header files, plus "LIMITED_API" defines.